interpreter/src/objects.rs

```diff
@@ -245,6 +280,29 @@ impl Ord for Value {
             (Value::Null, Value::Null) => Ordering::Equal,
             (Value::Duration(a), Value::Duration(b)) => a.cmp(b),
             (Value::Timestamp(a), Value::Timestamp(b)) => a.cmp(b),
+            // Allow different numeric types to be compared without explicit casting.
+            (Value::Int(a), Value::UInt(b)) => a
+                .to_owned()
+                .try_into()
+                .and_then(|a: u64| Ok(a.cmp(b)))
+                .unwrap_or(Ordering::Greater),
```

Comment:
Why does this one default to `Ordering::Greater` while the others default to `Ordering::Equal`?

Reply:
Oops, looks like I got it flipped actually, it should be `Ordering::Less`. If an i64 can't cast to a u64 it must be less than 0, considering that `u64::MAX` > `i64::MAX`. I'll add a comment to clarify while I'm fixing it